cf-agent/files_properties.c

```diff
@@ -48,32 +48,17 @@ static const char *const SKIPFILES[] =
 static bool ConsiderFile(const char *nodename, const char *path, struct stat *stat)
 {
     int i;
-    
-    if (stat != NULL)
-    {
-        if (S_ISLNK(stat->st_mode))
-        {
-            Log(LOG_LEVEL_INFO, "'%s' is a symbolic link", nodename);
-        }
-        else if (S_ISDIR(stat->st_mode))
-        {
-            Log(LOG_LEVEL_INFO, "'%s' is a directory", nodename);
-        }
-        
-        Log(LOG_LEVEL_VERBOSE, "'%s' has size %ld and full mode %o", nodename, (unsigned long) (stat->st_size),
-              (unsigned int) (stat->st_mode));
-    }
 
     if (strlen(nodename) < 1)
     {
-        Log(LOG_LEVEL_ERR, "Empty (null) filename detected in '%s'", path);
+        Log(LOG_LEVEL_ERR, "Empty (null) filename detected in '%s', skipping", path);
         return false;
     }
 
     if (stat != NULL && (S_ISREG(stat->st_mode) || S_ISLNK(stat->st_mode)) &&
         IsItemIn(SUSPICIOUSLIST, nodename))
     {
-        Log(LOG_LEVEL_ERR, "Suspicious file '%s' found in '%s'", nodename, path);
+        Log(LOG_LEVEL_ERR, "Filename '%s/%s' is classified as suspiscious, skipping", path, nodename);
```

Comment:
You are right! I agree with changing to WARN, but I'm a bit sceptical about skipping the files, since it's a change in behaviour.

I would say that skipping files would be better as a new "skipfiles" option that would populate the `SKIPFILES` array.